templates/shrine/src/shrine-detail.html

```diff
@@ -243,7 +254,8 @@
           <h1>[[item.title]]</h1>
           <p>[[item.description]]</p>
           <div class="quantitySelector">
-            <select>
+            <label class="sr-only" for="quantity">Number of items:</label>
+            <select id="quantity">
```

Comment:
@jsilvermist you mean about the use of the id in the select, don`t you? Is [one of the manners to use this element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label).